optimum/exporters/onnx/base.py

```diff
@@ -126,21 +126,22 @@ class OnnxConfig(ExportConfig, ABC):
     _TASK_TO_COMMON_OUTPUTS = {
```

Comment:
small nit: Since this is a refactor, could you also make the following list alphabetical? I guess the audio keys are only misaligned.